src/mongoose_account_api.erl

```diff
@@ -184,6 +185,30 @@ check_password_hash(JID, PasswordHash, HashMethod) ->
             {incorrect, "Password hash is incorrect"}
     end.
 
+-spec import_users(file:filename()) -> {ok, mongoose_import_users:summary()}
```

Comment:
I think the returned type is different, because we wrap each element in `{ok, ...}`